### DISCUSSION TITLE
Website: Duplicate content ?

### DISCUSSION THREAD

**Discussion by johndodev** [caprover/caprover#1426] [Q&A] [answered]
Hello,

I'm discovering caprover, and deploy a website with it, but, the website has an real domain, and a default caprover domain like "[app_name].captain.[root_caprover_domain]. So, it's technically 2 websites with the same content, which is a problem.

Is there any simple way to "disable" the default caprover domain ?

**Comment by githubsaturn** [comment, marked as answer]
https://github.com/caprover/caprover/issues/642
